docs/weblog/README.md

```diff
@@ -746,3 +752,13 @@ Query parameters required in the `GET` method:
 
 Examples:
 - `GET`: `/session/user?sdk_user=sdkUser`
+
+### \[GET\] /user/me
+
+A call to `/user/me` can be done in order to get the current user id logged in the system.
```

Comment:
should it return the user in the body ? that's not very clear from that description